setup.py

```diff
@@ -91,6 +91,12 @@ def run_tests(self):
       author='Chris Beaumont, Thomas Robitaille',
       author_email='glueviz@gmail.com',
       url='http://glueviz.org',
+      install_requires=['numpy', 'pandas', 'astropy', 'matplotlib'],
+      extras_require={
```

Comment:
It would be more consistent indeed but no, it is `extras_require` :
https://pythonhosted.org/setuptools/setuptools.html#declaring-extras